src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+const defaultState = {
+};
+
+const reducer = (state = defaultState, action) => {
+  return state;
+};
+
+export default reducer;
```

Comment:
please add an empty line as last line in every file (eslint)

Reply:
@ChristianSchmidt1981 
It has the empty line but it is not showing up.